src/components/BeaconChainActions.tsx

```diff
@@ -140,21 +108,30 @@ const BeaconChainActions: React.FC = () => {
   ]
 
   return (
-    <Container>
-      <StyledCardContainer>
-        <StyledCardLeft
+    <Box mb={16}>
+      <Flex flexDir={{ base: "column", md: "row" }} pt={4}>
+        <Box
+          as={Card}
+          w="full"
+          ml={0}
+          mr={{ base: 0, md: 4 }}
+          mb={{ base: 8, md: 0 }}
           emoji=":money_with_wings:"
           title={translateMessageId("consensus-become-staker", intl)}
           description={translateMessageId("consensus-become-staker-desc", intl)}
         >
-          <StyledButtonLink to="https://launchpad.ethereum.org">
+          <ButtonLink mb={3} to="https://launchpad.ethereum.org">
             <Translation id="get-started" />
-          </StyledButtonLink>
+          </ButtonLink>
           <ButtonLink variant="outline" to="/staking/">
             <Translation id="page-upgrades-index-staking-learn" />
           </ButtonLink>
-        </StyledCardLeft>
-        <StyledCardRight
+        </Box>
+        <Box
+          as={Card}
```

Comment:
`Card` is already exporting a chakra component. Lets refactor a bit the `Card` component to spread the receiving props:
```tsx
// in Card.tsx
const Card: React.FC<IProps> = ({...rest}) => (
  <Flex ... {...rest}>
```

```suggestion
        <Card
```

Reply:
Resolved